kvmagent/kvmagent/plugins/ceph_storage_plugin.py

```
@@ -166,20 +167,11 @@ def _is_luks_rbd(self, install_path, conf_path):
-                return None
-            # rbd du json returns a list; first row is the image itself when no
-            # snapshots are queried. Pick the first numeric used_size_ we see.
-            for image_usage in images:
-                used = getattr(image_usage, "used_size_", None)
-                if used is not None:
-                    return long(used)
-            return None
+            out = shell.call("/usr/bin/qemu-img info --output=json %s" %
+                             CephStoragePlugin._rbd_uri(install_path, conf_path))
+            return json.loads(out).get("actual-size")
         except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)